app/src/main/java/doit/study/droid/Utils/Sound.java

```diff
@@ -1,4 +1,4 @@
-package doit.study.droid.utils;
+package doit.study.droid.Utils;
```

Comment:
Please revert to the initial lower case.
Kotlin's naming conversions https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules